### DISCUSSION TITLE
Changing file permissions from within nnn?

### DISCUSSION THREAD

**Discussion by Logicdemon** [jarun/nnn#1799] [General]
Hi, would someone kindly tell me how I can change file permissions from within nnn? I figured out pressing * will change a file from/too executable 755 -rw-r--r--  However I'm stuck trying to set some files to 600 -r------- I can easily do it via the command line but would be nice to be able to do it via nnn.

Cheers

**Comment by jarun** [comment]
Please write a script, add a keybind and run the command like a plugin.

https://github.com/jarun/nnn/tree/master/plugins#running-commands-as-plugin-

**Comment by N-R-K** [comment]
> I can easily do it via the command line

You can use `!` to drop down to a temporary shell.
